src/snowcli/app/cli_app.py

```diff
@@ -88,6 +89,16 @@ def _version_callback(value: bool):
         raise typer.Exit()
 
 
+@_do_not_execute_on_completion
+@_commands_registration.after
+def _options_structure_callback(value: bool):
+    if value:
+        ctx = click.get_current_context()
+        output_json = generate_commands_structure(ctx.command).print_with_options()
```

Comment:
output_json variable is unused, IMO you don't need it

Reply:
Done